src/qhttp/Response.h

```diff
@@ -75,7 +75,12 @@ class Response : public std::enable_shared_from_this<Response> {
              DoneCallback const& doneCallback, std::size_t const maxResponseBufSize);
 
     std::string _headers() const;
-    void _startTransmit();
+
+    //----- Mark a start of transmission. Return 'true' if multiple transmission attempts
+    //      were detected. Duplicate transmission requests will be ignored. An error message
+    //      may be logged. The ongoing transmission won't be affected.
+
+    bool _startTransmit();
```

Comment:
Perhaps reverse the sense of this boolean for better comprehensibility?  (`true` usually implies success?)

Thank you for remaining consistent with my quirky comment style :-)

Reply:
Good suggestion. Fixed. Tested.